lib/tool_shed/util/shed_util_common.py

```diff
@@ -825,14 +820,10 @@ def get_tool_shed_status_for_installed_repository( app, repository ):
         # using a boolean value.
         log.debug( "Error attempting to get tool shed status for installed repository %s: %s\nAttempting older 'check_for_updates' method.\n" %
                    ( str( repository.name ), str( e ) ) )
-        params = '?name=%s&owner=%s&changeset_revision=%s&from_update_manager=True' % ( str( repository.name ),
-                                                                                        str( repository.owner ),
-                                                                                        str( repository.changeset_revision ) )
-        url = common_util.url_join( tool_shed_url,
-                                    'repository/check_for_updates%s' % params )
```

Comment:
Here you are re-using `params` as defined above, but you seem to be missing `from_update_manager=True`.